### PR TITLE
1256 rx desktop only

### PR DESCRIPTION
**Pull request by lealmendozaalexander**
**Note**: Delete the description statements, complete each step. **None are optional**, but can be justified as to why they cannot be completed as written. Provide known gaps to testing that may raise the risk of merging to production.

## Summary

- _(Summarize the changes that have been made to the platform)_
Add new library that will detect if user is on a mobile device because of accessibility issues with mobile devices specifically screen readers
- _(If bug, how to reproduce)_
- _(What is the solution, why is this the solution)_
- _(Which team do you work for, does your team own the maintenance of this component?)_
Virtual Agent Platform, we own this component 
- _(If using a flipper, what is the end date of the flipper being required/success criteria being targeted)_

## Related issue(s)
https://github.com/department-of-veterans-affairs/va-virtual-agent/issues/1256

## Testing done

- _Describe what the old behavior was prior to the change_
Before user was able to go into prescription chat skill regardless if they were on mobile or not 
- _Describe the steps required to verify your changes are working as expected_
If you were to try to enter the prescriptions skill on a mobile device you should get an error message saying this feature is not available
- _Describe the tests completed and the results_
Created test to check if the isMobile variable is being sent to PVA 
- _Exclusively stating 'Specs and automated tests passing' is NOT acceptable as appropriate testing
- _Optionally, provide a link to your [test plan](https://depo-platform-documentation.scrollhelp.site/developer-docs/create-a-test-plan-in-testrail) and [test execution records](https://depo-platform-documentation.scrollhelp.site/developer-docs/execute-tests-in-testrail)_

## Screenshots

_Note: This field is mandatory for UI changes (non-component work should NOT have screenshots)._

|         | Before | After |
| ------- | ------ | ----- |
| Mobile  |        |     |
| Desktop |        |       |

## What areas of the site does it impact?

N/A

### Quality Assurance & Testing

- [x] I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x] Linting warnings have been addressed
- [ ] Documentation has been updated ([link to documentation](#) \*if necessary)
- [ ] Screenshot of the developed feature is added
- [ ] [Accessibility testing](https://depo-platform-documentation.scrollhelp.site/developer-docs/wcag-2-1-success-criteria-and-foundational-testing) has been performed

### Error Handling

- [x] Browser console contains no warnings or errors.
- [ ] Events are being sent to the appropriate logging solution
- [ ] Feature/bug has a monitor built into Datadog or Grafana (if applicable)

### Authentication

- [ ] Did you login to a local build and verify all authenticated routes work as expected with a test user

### :warning: Team Sites (only applies to modifications made to the VA.gov header) :warning:

- [ ] The vets-website header does not contain any web-components
- [ ] I used the [proxy-rewrite steps](https://github.com/department-of-veterans-affairs/vets-website/tree/main/src/applications/proxy-rewrite#local-dev) to test the injected header scenario
- [ ] I reached out in the `#sitewide-public-websites` Slack channel for questions

## Requested Feedback

(OPTIONAL) _What should the reviewers know in addition to the above. Is there anything specific you wish the reviewer to assist with. Do you have any concerns with this PR, why?_

### CODE REVIEW

**Review by powellkerry — APPROVED** [review]

**Review by eubiera — APPROVED** [review]
LGTM

File: src/applications/virtual-agent/tests/components/webchat/startConvoAndTrackUtterances.unit.spec.js [hunk 1]
```diff
@@ -367,5 +367,21 @@ describe('makeBotStartConvoAndTrackUtterances actions', () => {
...
     });
+    it('should pass isMobile to PVA', async () => {
+      await StartConvoAndTrackUtterances.makeBotStartConvoAndTrackUtterances(
+        'csrfToken',
+        'apiSession',
+        'apiURL',
+        'baseURL',
+        'userFirstName',
+        'userUuid',
+        'isMobile',
```
Comment (ksk385) [inline src/applications/virtual-agent/tests/components/webchat/startConvoAndTrackUtterances.unit.spec.js:378]: Wouldn't this be a boolean value as opposed to a string?

Comment (aaronyoung-tw) [reply src/applications/virtual-agent/tests/components/webchat/startConvoAndTrackUtterances.unit.spec.js:378]: thats just a test that whatever we put in is passed through - we could have done a boolean, but the same test applies

Comment (ksk385) [reply src/applications/virtual-agent/tests/components/webchat/startConvoAndTrackUtterances.unit.spec.js:378]: Its good practice to make the tests reflect the real data types of production code - this serves as documentation of how the code works on a deeper level.

Comment (aaronyoung-tw) [reply src/applications/virtual-agent/tests/components/webchat/startConvoAndTrackUtterances.unit.spec.js]: We have updated the code to reflect as such

**Pull request merged**